.gitignore

```diff
@@ -25,10 +25,33 @@ doc/api/
 
 .flutter-plugins
 .flutter-plugins-dependencies
+.vs/*
+.vscode/
 .vs*
 */ios/
 linux/
 macos/
 web/
 windows/
 Flutter/
+teamA/linux/flutter/generated_plugin_registrant.cc
```

Comment:
Very small comment but these could be reduced to teamA/linux/* and so on, that just keeps the clutter down.

Reply:
Done.